using-mobile-extensions/adobe-campaignclassic/adobe-campaignclassic-api-reference.md

```diff
@@ -23,7 +23,7 @@ String campaignClassicExtensionVersion = CampaignClassic.extensionVersion();
 
 {% tab title="iOS (AEP 3.x)" %}
 
-This extension is not compatible with the AEPCore extension at this time, for more details see [Current SDK Versions](https://aep-sdks.gitbook.io/docs/resources/upgrading-to-aep/current-sdk-versions#ios-swift)
+This extension is not compatible with the AEPCore extension at this time, for more details see [Current SDK Versions](../../resources/upgrading-to-aep/current-sdk-versions#ios-swift)
```

Comment:
Should this be `../../resources/upgrading-to-aep/current-sdk-versions.md#ios-swift`?

Reply:
updated the message regarding AEP 3.x campaign classic. The linked page didn't give much info.